cmd/menu.go

```diff
@@ -0,0 +1,186 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sync"
+
+	"github.com/jroimartin/gocui"
+	"github.com/urfave/cli/v2"
+)
+
+var menuItems = []string{
+	Archon.String(),
+	//Patcher.String(),
+	//GenerateServerCertificates.String(),
+	//RunPacketAnalyzer.String(),
+	//RunAccountTool.String(),
```

Comment:
These don't work for two reasons:
- they need to be converted to the CLI package
- the ones that take user input will take some effort to get the GUI to accept stdin

I do have them all converted but they'll fail.